httpbin/helpers.go

```diff
@@ -133,6 +133,13 @@ func parseBody(w http.ResponseWriter, r *http.Request, resp *bodyResponse) error
 
 	ct := r.Header.Get("Content-Type")
 	switch {
+	// cases where we don't need to parse the body
+	case ct == "":
+		fallthrough
```

Comment:
I think `ct == ""` should be treated as if `ct == "application/octet-stream"` (i.e., if no content type is given, just treat the body as opaque bytes).

Reply:
I had it set up like that first, but then a lot of tests failed, that resulted in this test failing: https://github.com/mccutchen/go-httpbin/blob/main/httpbin/handlers_test.go#L659

Reply:
I pushed a second commit, that implements this change, I adapted that one test.